tests/test_doctest.py

```diff
@@ -28,11 +30,14 @@ def check_output(self, want: str, got: str, optionflags: int) -> bool:
 finder = doctest.DocTestFinder(exclude_empty=True)
 
 
-@pytest.mark.parametrize('test', finder.find(deal._aliases))
+@pytest.mark.parametrize('test', chain(
+    finder.find(deal._aliases),
+    finder.find(deal._testing),
+))
 def test_doctest(test):
     state.color = False
     try:
-        runner = doctest.DocTestRunner(checker=Checker())
+        runner = doctest.DocTestRunner(checker=Checker(), optionflags=doctest.ELLIPSIS)
```

Comment:
It would be possible, if ugly, to use [the `setUp` argument](https://docs.python.org/3/library/doctest.html#doctest.DocTestSuite) to make `import atheris` work, if you think that monkeypatching is worth it for the docs.